app/scripts/modules/google/src/image/ImageSelect.tsx

```diff
@@ -0,0 +1,61 @@
+import { module } from 'angular';
+
+import { chain } from 'lodash';
+import * as React from 'react';
+import { Async, AutocompleteResult, Option } from 'react-select';
+import { react2angular } from 'react2angular';
+
+interface IImage {
+  imageName: string;
+}
+
+interface IImageSelectProps {
+  availableImages: IImage[];
+  selectedImage: string;
+  selectImage: (image: string, target?: any) => void;
+  target?: any;
+}
+
+export class ImageSelect extends React.Component<IImageSelectProps> {
+  private loadOptions = (inputValue: string): Promise<AutocompleteResult<string>> => {
+    return new Promise(resolve => {
+      if (!inputValue || inputValue.length < 3) {
+        resolve({
+          options: [],
+          complete: false,
+        });
+      } else {
+        const filteredOptions = chain(this.props.availableImages)
+          .filter(i => i.imageName.toLowerCase().includes(inputValue))
```

Comment:
Does `inputValue` need to also be lower-cased (or does that happen somewhere prior to this function)?

Reply:
It happens mAgIcAlLy at some point before this callback receives the input value. This Async select component has an `ignoreCase` prop that defaults to true. I'm setting the also-defaulted-to-true `ignoreAccents` prop to false to [improve perf](https://github.com/JedWatson/react-select/issues/3128#issuecomment-487256349).

Reply:
Awesome, thanks for the explanation!